modules/page-edit.php

```diff
@@ -10,6 +10,46 @@
 		global $settings, $env;
 		
 		/**
+		 * @api {post} ?action=preview_edit&page={pageName}[&newpage=yes]	Get a preview of the page
+		 * @apiDescription	Gets a preview of the current edit state of a given page
+		 * @apiName 		PreviewPage
+		 * @apiPermission	Anonymous
+		 * 
+		 * @apiUse	PageParameter
+		 * @apiParam	{string}	newpage 	Set to 'yes' if a new page is being created.

```

Comment:
You haven't included your `preview-edit` parameter here? This is useful because then when you rebuild the docs with `make` (or just with apidoc if you look at the command in the makefile), it parses these blocks to make an [api documentation page](http://sbrl.github.io/Pepperminty-Wiki/RestApiDocs/).